### ISSUE TITLE
Breaks doxygen search

### ISSUE THREAD

**Issue by Ede123** [Cimbali/CleanLinks#132]
Maybe I'm too stupid, but I can't seem to manage to properly whitelist the following:
* Visit https://inkscape.gitlab.io/inkscape/doxygen/index.html
* Click into the search field to the upper right
* Enter "pathvector":<br>![grafik](https://user-images.githubusercontent.com/904570/108626837-9a4fe680-7452-11eb-8861-d56279f78125.png)
* Click on `Geom::PathVector`:<br>![grafik](https://user-images.githubusercontent.com/904570/108626883-cc614880-7452-11eb-9e09-b8fcc73ffb15.png)
* Result: Page opens in the tiny frame of the search box instead of the actual iframe target (`_parent`).

I've set a whitelist rule (see below), but it does not seem to work. Disabling the CleanLinks extensions completely fixes the issue however.
![grafik](https://user-images.githubusercontent.com/904570/108626971-501b3500-7453-11eb-955d-ae2c1bda13e7.png)

**Comment by Cimbali** [comment]
The protocol is not needed in the URL matching. That means just `inkscape.gitlab.io` not `https://inkscape.gitlab.io`.

Maybe we can detect it in the future and either ignore it or setup a specific error message.

**Comment by Ede123** [comment]
Yep, I realized that just after posting the issue (would be nice if there was something to catch this; I assume this would break subdomain matching and protocol changes or would it even break altogether?).

However, eve. after dropping the protocol, the whitelist rule still does not solve the issue for me. Could you make it work on your end? (The URL is publicly reachable, so you can cross-check from your end if the error is in front or behind the display ;-) ).

**Comment by Cimbali** [comment]
Interesting, this seems to come from the `target="_parent"`. In the Clean Links preferences you should tick the box “Allow links to different tabs (target attribute)”.

Maybe we could also investigate allowing _parent but not _blank or more fine-grain options like that.

**Comment by Ede123** [comment]
Thanks, that did the trick (even without a whitelist entry).

What's the purpose of this prefernece? Usually it makes sense if people use `target="_blank"` and obviously even more so if the `target` attribute is used for iframe navigation like in this case, so even if you'd mage it more fine-grained, I'd likely keep it set to allow all of them.

**Comment by Cimbali** [comment]
I don't use that option personally, but it's used to enforce single-tab browsing. I think it's one of those legacy features, like catching text links, that a few users still use. 

IMHO nowadays Firefox does a good job of forcing _blank targets into tabs etc. rather than creating new windows, and possibly this feature is also less misused than a couple of years ago (or I stopped visiting sites that abuse it).

Anyways I just checked and this option is off on fresh installs, so I don't think it should be too harmful.
